WTWSMS/history/characters/WTWSMS_enlil.txt

```diff
@@ -7594,10 +7594,10 @@
 	dynasty=1042522
 	religion="nicene"
 	culture="romano_hispanic"
-	540.1.1={
+	520.1.1={
 		birth = yes
 	}
-	610.1.1={
+	572.1.1={
 		death = yes
```

Comment:
Why did you change these dates?

Reply:
No, we discussed some other character, not this one. 36558 is not a child while holding territories in the main branch.

Reply:
No, we discussed some other character, not this one. 36558 is not a child while holding territories in the main branch.

Reply:
Should I change it back? I think it was useful to avoid having a holder = 0 in the county, That´s why he dies when the duchy dissappears.

Reply:
The birth date was simply to avoid having a very old ruler

Reply:
There should definitely not be holder = 0 entry, although isn't it possible to give it to the liege? He will just stay a landless pretender as he is currently.

Reply:
Well, That could be done, but is the current setup that bad?

Reply:
I don't know what the setup becomes with this, I was speaking with the setup in the master branch in mind.

Reply:
This age change doesn´t acutally change the setup, simply makes an unlanded character dissappear. Master wouldn´t be affected by anything.

Reply:
Okay, fair enough.